stdlib/universe/schema_functions_test.go

```diff
@@ -876,6 +876,60 @@ func TestDropRenameKeep_Process(t *testing.T) {
 				),
 			}},
 		},
+		{
+			name: "duplicate group key",
+			spec: &universe.SchemaMutationProcedureSpec{
+				Mutations: []universe.SchemaMutation{
+					&universe.DuplicateOpSpec{
+						Column: "1a",
+						As:     "1a_1",
+					},
+				},
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "1a", Type: flux.TFloat},
+					{Label: "2a", Type: flux.TFloat},
+					{Label: "3a", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{1.0, 2.0, 3.0},
+					{1.0, 12.0, 3.0},
+					{1.0, 22.0, 3.0},
+				},
+				KeyCols:   []string{"1a", "3a"},
+				KeyValues: []interface{}{1.0, 3.0},
+				GroupKey: execute.NewGroupKey(
```

Comment:
Likewise, the `executetest.ProcessTestHelper` infrastructure should create the `GroupKey` for you, so you don't need to define it here.  For table that has data in it, on `KeyCols` needs to be defined.

Reply:
Great, thank you. It was copy paste from other tests in the same file. Anyhow, thanks for the clarification!